### DISCUSSION TITLE
Question about NAT Traversal

### DISCUSSION THREAD

**Discussion by joseph1267** [strongswan/strongswan#935] [Q&A] [answered]
In IKE SA INIT, when Initiator has NAT_DETECTION_SOURCE_IP and NAT_DETECTION_DESTINATION_IP, but Responder do not have both NAT_DETECTION_SOURCE_IP and NAT_DETECTION_DESTINATION_IP in the IKE SA INIT response, what should be the proper action for strongswan? 

Thank you

**Comment by tobiasbrunner** [comment, marked as answer]
Disabling NAT-T.
